stream_reader_test.go

```diff
@@ -75,4 +75,16 @@ func TestStreamReaderRecvRaw(t *testing.T) {
 	if !bytes.Equal(rawLine, []byte("{\"key\": \"value\"}")) {
 		t.Fatalf("Did not return raw line: %v", string(rawLine))
 	}
+
+	// issue: https://github.com/sashabaranov/go-openai/issues/781
```

Comment:
No need for this comment, `git blame` will point to this PR/issue anyways

Reply:
ok, got it~